target_chains/solana/Anchor.toml

```diff
@@ -1,15 +1,18 @@
 [features]
 seeds = false
 skip-lint = false
-[programs.localnet]
-solana_receiver = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
+
+[programs.devnet]
+pyth_solana_receiver = "5dXnHcDXdXaiEp9QgknCDPsEhJStSqZqJ4ATirWfEqeY"
```

Comment:
Can you grind a key using `solana-keygen grind --starts-with`. This key will be shared in all networks and will be added

Reply:
Also need to check magic number as in https://github.com/pyth-network/pyth-crosschain/blob/main/governance/remote_executor/programs/remote-executor/src/lib.rs#L77